modules/admin_manual/pages/enterprise/security/oidc/index.adoc

```diff
@@ -0,0 +1,118 @@
+= OpenID Connect (OIDC)
+:toc: right
```

Comment:
You need to add a toclevel: 2 directive
(Reason, because you have important subsections, see below)

Reply:
Thanks for spotting that.